test/flows/uk_benefits_abroad_flow_test.rb

```diff
@@ -0,0 +1,1302 @@
+            assert_next_node :employer_paying_ni?, for_response: "australia"
+          end
+        end
+
+        context "winter_fuel_payment" do
+          setup do
+            add_responses which_benefit?: "winter_fuel_payment"
+          end
+
+          should "have next node of is_british_or_irish? for an 'ireland' response" do
+            assert_next_node :is_british_or_irish?, for_response: "ireland"
+          end
+
+          should "have a next node of worked_in_eea_or_switzerland? for any EEA country response except ireland" do
+            assert_next_node :worked_in_eea_or_switzerland?, for_response: "liechtenstein"
+          end
+
+          should "have a next node of wfp_not_eligible_outcome for any non-EEA country response" do
+            assert_next_node :wfp_not_eligible_outcome, for_response: "australia"
+          end
+        end
+
+        context "child_benefit" do
+          setup do
+            add_responses which_benefit?: "child_benefit"
+          end
+
+          should "have a next node of do_either_of_the_following_apply? for any EEA country response" do
+            assert_next_node :do_either_of_the_following_apply?, for_response: "liechtenstein"
+          end
+
+          should "have a next node of child_benefit_fy_going_abroad_outcome for any social security country with child benefit response and if going_abroad" do
+            assert_next_node :child_benefit_fy_going_abroad_outcome, for_response: "north-macedonia"
+          end
+
+          should "have a next node of child_benefit_fy_already_abroad_outcome for any social security country with child benefit response and if already_abroad" do
+            add_responses going_or_already_abroad?: "already_abroad"
+            assert_next_node :child_benefit_fy_already_abroad_outcome, for_response: "north-macedonia"
+          end
+
+          %w[barbados canada guernsey israel jersey mauritius new-zealand].each do |country|
+            should "have a next node of child_benefit_ss_outcome for a '#{country}' response" do
+              assert_next_node :child_benefit_ss_outcome, for_response: country
+            end
+          end
+
+          %w[jamaica turkey usa].each do |country|
+            should "have a next node of child_benefit_jtu_outcome for a '#{country}' response" do
+              assert_next_node :child_benefit_jtu_outcome, for_response: country
+            end
+          end
+
+          should "have a next node of child_benefit_not_entitled_outcome for any other country response" do
+            assert_next_node :child_benefit_not_entitled_outcome, for_response: "australia"
+          end
+        end
+
+        context "iidb" do
+          setup do
+            add_responses which_benefit?: "iidb",
+                          iidb_already_claiming?: "yes"
+          end
+
+          should "have next node of iidb_going_abroad_eea_outcome for any response in EEA but not in social security country with iidb if going_abroad" do
```

Comment:
```suggestion
          should "have next node of iidb_going_abroad_eea_outcome for any response in an EEA country if going_abroad" do
```

Since the code is an OR it shouldn't matter whether the country is in the social-security-country-with-iidb-group or not